### PR TITLE
doc(examples) add hybrid enterprise example values

### LINKED ISSUE #202: need Enterprise examples for Hybrid cp and dp values.yaml files
In addition to the OSS hybrid examples, would be helpful to have minimal sample cp and dp values.yaml files to start with.

### PR DESCRIPTION
**Pull request by PhilKuer**
#### What this PR does / why we need it:
 -  add hybrid enterprise example values

#### Which issue this PR fixes
  - fixes #202 
 
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] PR is based off the current tip of the `main` branch.
- [x] Changes are documented under the "Unreleased" header in CHANGELOG.md
- [x] New or modified sections of values.yaml are documented in the README.md
- [x] Commits follow the [Kong commit message guidelines](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#commit-message-format)

### CODE REVIEW

File: charts/kong/example-values/minimal-kong-enterprise-hybrid-control.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    enabled: true
+    servicePort: 8001
+    containerPort: 8001
+
+cluster:
+  enabled: true
+  tls:
+    enabled: true
+    servicePort: 8005
+    containerPort: 8005
```
Comment (rainest) [inline charts/kong/example-values/minimal-kong-enterprise-hybrid-control.yaml]: The ports are unchanged from the defaults so shouldn't be set here also:

```suggestion
admin:
  enabled: true
  http:
    enabled: true

cluster:
  enabled: true
  tls:
    enabled: true
```

File: charts/kong/example-values/minimal-kong-enterprise-hybrid-control.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+#   that are sent to both the admin API and Kong Manager, and any Dev Portal
+#   instances with authentication must create cookies that are sent to both
+#   the Portal and Portal API.
+
+image:
+  repository: kong/kong-gateway
+  tag: "2.8"
+
+env:
+  prefix: /kong_prefix/
```
Comment (rainest) [inline charts/kong/example-values/minimal-kong-enterprise-hybrid-control.yaml]: Included in the default values.
```suggestion
```

File: charts/kong/example-values/minimal-kong-enterprise-hybrid-control.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  enabled: false
+
+secretVolumes:
+- kong-cluster-cert
+
+postgresql:
+  enabled: true
+  auth:
+    username: kong
+    database: kong
```
Comment (rainest) [inline charts/kong/example-values/minimal-kong-enterprise-hybrid-control.yaml]:
```suggestion
```

File: charts/kong/example-values/minimal-kong-enterprise-hybrid-control.yaml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+enterprise:
+  enabled: true
+  # See instructions regarding enterprise licenses at https://github.com/Kong/charts/blob/master/charts/kong/README.md#kong-enterprise-license
+  license_secret: kong-enterprise-license # CHANGEME
+  vitals:
+    enabled: false
+  portal:
+    enabled: false
+  rbac:
+    enabled: false
```
Comment (rainest) [inline charts/kong/example-values/minimal-kong-enterprise-hybrid-control.yaml]:
```suggestion
```

File: charts/kong/example-values/minimal-kong-enterprise-hybrid-data.yaml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+# Hybrid mode requires a certificate. See https://github.com/Kong/charts/blob/main/charts/kong/README.md#certificates
+# to create one.
+
+
+image:
+  repository: kong/kong-gateway
+  tag: "2.8"
+
+env:
+  prefix: /kong_prefix/
```
Comment (rainest) [inline charts/kong/example-values/minimal-kong-enterprise-hybrid-data.yaml]:
```suggestion
```

File: charts/kong/example-values/minimal-kong-enterprise-hybrid-data.yaml [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+enterprise:
+  enabled: true
+  # See instructions regarding enterprise licenses at https://github.com/Kong/charts/blob/master/charts/kong/README.md#kong-enterprise-license
+  license_secret: kong-enterprise-license # CHANGEME
+  vitals:
+    enabled: false
+  portal:
+    enabled: false
+  rbac:
+    enabled: false
```
Comment (rainest) [inline charts/kong/example-values/minimal-kong-enterprise-hybrid-data.yaml]:
```suggestion
  portal:
    enabled: false
  rbac:
    enabled: false
```

File: charts/kong/CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* doc(examples) add hybrid enterprise example values
+  ([575](https://github.com/Kong/charts/pull/575))
+
```
Comment (rainest) [inline charts/kong/CHANGELOG.md]:
```suggestion
```

**Review by rainest — CHANGES_REQUESTED** [review]
Mostly stripping values that are already present in the default values.yaml.

Removed the changelog since it was causing a merge conflict and we don't usually cover docs changes there anyway.

**Comment by PhilKuer** [comment]
Thanks for your feedback.

I have stripped both example-values to match the default values. Changelog line is also removed.

**Review by rainest — APPROVED** [review]

**Pull request merged**